### PR TITLE
Added Styling to Project

### PR DESCRIPTION
**Pull request by BryteLitty**
In this PR, I added styling to my project to help improve the user interactivity and experience

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  font-family: "Montserrat", sans-serif;
+}
+
+body {
+  background-color: #f5f6fa;
+}
+
+.app {
+  margin: 4%;
+}
```
Comment (jasonfelice) [inline src/App.css]: - [ ] This margin is unnecessary. Kindly remove this margin. This way, everything will fit into the viewport.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
+
+* {
+  font-family: "Montserrat", sans-serif;
+}
```
Comment (jasonfelice) [inline src/App.css]: - [ ] Please reset the margin by using `margin: 0` to remove spaces from the sides. 😸

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @BryteLitty, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No Linter errors✔️
Good Design✔️
Followed GitFlow✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @BryteLitty, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Made requested changes ✔
- PR ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**